docs/program/extend/modular-resources/examples/add-tflite-module.md

```diff
@@ -0,0 +1,321 @@
+---
+title: "Add a TensorFlow Lite Modular Service"
+linkTitle: "Add a TensorFlow Lite Modular Service"
+weight: 70
+type: "docs"
+description: "Add a MLModel modular-resource-based service which uses TensorFlow Lite to classify audio samples."
+tags: ["ml", "model training", "services"]
+# SMEs: Andrew Morrow
+---
+
+Viam provides an example [modular resource](extend/modular-resources) written in C++ that extends the [ML model](/services/ml/) service to run any TensorFlow Lite model.
```

Comment:
I think it is worth adding a sentence here to make extra clear that the module is an example, and what the purpose of the example is. Something like: `The example is intended to demonstrate the principles of operation of implementations of MLModel servers and clients, in order to provide a roadmap for users to implement their own. The components used here are for example purposes only and are not intended for production use.` Feel free to adjust to your liking but that's the content I'm hoping to see included.